packages/manager-react-components/src/hooks/date/useFormattedDate.ts

```diff
@@ -0,0 +1,46 @@
+import { useTranslation } from 'react-i18next';
+
+export const defaultUnknownDateLabel = 'N/A';
```

Comment:
shouldn't  that label be translated?

Reply:
Well actually I put this to have a default value without translation and we can provide a translated one via the parameters.

But maybe we should return undefined in this case and let the app manage the absence of value

Or have a value that works for every project, I don't know.